### ISSUE TITLE
ignition: error at line 1, no filesystem specified

### ISSUE THREAD

**Issue by th3penguinwhisperer** [openshift/installer#3030]
# Version

Tried with multiple versions from the git repository (branch release-4.2, release-4.3, release-4.4, release-4.5)

```console
$ openshift-install unreleased-master-2457-ge1a6a984b73a818a48f800eecc05ebbc526fc5b9
built from commit e1a6a984b73a818a48f800eecc05ebbc526fc5b9
release image registry.svc.ci.openshift.org/origin/release:4.3
```

This is done on a CentOS 7.7 system.

I've tried all day and can't seem to get this working. I had expected with the RHCOS it would have little things to go wrong.

# Platform:
libvirt

# What happened?
I see 2 machines being created: the bootstrap and a master machine. The bootstrap is ssh reachable and I see output in the journalctl. It takes a while for the port 22623 becoming available for the master.

When I check on the master console I see it trying to go to the bootstrap on port 22623.
After a while the file is retrieved and then I get immediately errors like these:
```
ignition[123]: error at line 1, column 791
no filesystem specified
ignition[123]: error at line 1, column 1191
no filesystem specified
...
```

```
INFO Obtaining RHCOS image file from 'https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.4/44.81.202001241431.0/x86_64/rhcos-44.81.202001241431.0-qemu.x86_64.qcow2.gz?sha256=56154a5c68e94879ff276e9ad6a7efc080cb0cb99b034f44e1bcd41a371f6878' 
INFO The file was found in cache: /root/.cache/openshift-installer/image_cache/893b690279d58ad4a014862439fd1b95. Reusing... 
INFO Creating infrastructure resources...         
INFO Waiting up to 20m0s for the Kubernetes API at https://api.oc4.mynet.local:6443... 
ERROR Attempted to gather ClusterOperator status after installation failure: listing ClusterOperator objects: the server could not find the requested resource (get clusteroperators.config.openshift.io) 
INFO Pulling debug logs from the bootstrap machine 
ERROR Attempted to gather debug logs after installation failure: failed to create SSH client, ensure the proper ssh key is in your keyring or specify with --key: dial tcp: lookup oc4-z2jf8-bootstrap.oc4.mynet.local: no such host 
FATAL Bootstrap failed to complete: waiting for Kubernetes API: context deadline exceeded
```

# What you expected to happen?

An openshift cluster being deployed (v4.x)

# How to reproduce it (as minimally and precisely as possible)?

I followed the instructions on libvirt/KVM setup (it was already setup but needed minor changes according to the doc).
install go
git clone the repo
git checkout release-4.x
TAGS=libvirt hacks/build.sh
openshift-install create cluster
Filled in the prompts.

# References

- https://github.com/openshift/installer/blob/master/docs/dev/libvirt/README.md

/label platform/libvirt

**Comment by th3penguinwhisperer** [comment]
[openshift_install.log](https://github.com/openshift/installer/files/4136211/openshift_install.log)

**Comment by abhinavdahiya** [comment]
when you build from master, please override the release image with `OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE` env to test OCP...
like `registry.svc.ci.openshift.org/ocp/release:4.3`

**Comment by th3penguinwhisperer** [comment]
I've tried this but got a different issue now:
- The webservice on the bootstrap node never comes up.
- The bootkube doesn't start because crio-configure doesn't start because release-image.service doesn't start.
- Running /usr/local/bin/release-image-download.sh shows the issue:
```
Pulling registry.svc.ci.openshift.org/ocp/release:4.3...
Error: error pulling image "registry.svc.ci.openshift.org/ocp/release:4.3": unable to pull registry.svc.ci.openshift.org/ocp/release:4.3: unable to pull image: Error initializing source docker://registry.svc.ci.openshift.org/ocp/release:4.3: Error reading manifest 4.3 in registry.svc.ci.openshift.org/ocp/release: unauthorized: authentication required
Pull failed. Retrying registry.svc.ci.openshift.org/ocp/release:4.3...
```
- I've checked the config.json and there's no auth token for this registry.
- So the master doesn't even get to the point where it gets the config.

I assume this is some kind of a workaround? I just want to basically install the same as would be on VMware vSphere so it feels odd having to use a different repo.
Am I blocked from using release-42 release-43 ... and can only use master?

If this is not a workaround it would be good the docs are updated (https://github.com/openshift/installer/blob/master/docs/dev/libvirt/README.md) as there's no way to figure this out.
